.github/workflows/int-and-connected-test-run.yml

```diff
@@ -158,138 +90,98 @@ jobs:
-      - name: "Build docker container via Gradle"
+      - name: Build docker container via Gradle
         uses: broadinstitute/datarepo-actions/actions/main@0.74.0
+        env:
+          # unset the GitHub Action default JAVA_HOME to build with JDK 17
+          JAVA_HOME:
         with:
           actions_subcommand: 'gradlebuild' # creates gcr build with git_hash tag
           sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-      - name: "Deploy to cluster with Helm"
+      - name: Deploy to cluster with Helm
         uses: broadinstitute/datarepo-actions/actions/main@0.74.0
         with:
           actions_subcommand: 'helmdeploy'
           helm_create_secret_manager_secret_version: 0.0.8
-          helm_datarepo_api_chart_version: 0.0.739
-          helm_datarepo_ui_chart_version: 0.0.360
+          helm_datarepo_api_chart_version: 0.0.744
+          helm_datarepo_ui_chart_version: 0.0.362
           helm_gcloud_sqlproxy_chart_version: 0.19.13
           helm_oidc_proxy_chart_version: 0.0.44
           sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-      - name: "Fetch gitHash for deployed integration version"
+      - name: Fetch gitHash for deployed integration version
         id: configuration
         run: |
           git_hash=$(git rev-parse --short HEAD)
           echo "git_hash=${git_hash}" >> $GITHUB_OUTPUT
           echo "Latest git hash for this branch: $git_hash"
-      - name: "Wait for deployment to come back online"
+      - name: Wait for deployment to come back online
         uses: broadinstitute/datarepo-actions/actions/wait-for-deployment@0.74.0
         timeout-minutes: 20
         env:
           DESIRED_GITHASH: ${{ steps.configuration.outputs.git_hash }}
           DEPLOYMENT_TYPE: 'api'
-      - name: "Run Test Runner smoke tests via Gradle"
+      - name: Run test runner smoke tests via Gradle
         uses: broadinstitute/datarepo-actions/actions/main@0.74.0
+        env:
+          # unset the GitHub Action default JAVA_HOME to run with JDK 17
+          JAVA_HOME:
+          # output plain logs instead of json
+          TDR_LOG_APPENDER: 'Console-Standard'
         with:
           actions_subcommand: 'gradletestrunnersmoketest'
           sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-      - name: "Run integration tests via Gradle"
+      - name: Run integration tests via Gradle
         uses: broadinstitute/datarepo-actions/actions/main@0.74.0
+        env:
+          # required for azure tests
+          AZURE_CREDENTIALS_APPLICATIONID: 0e29ec36-04e8-44d5-ae7c-50dc15135571
+          AZURE_CREDENTIALS_HOMETENANTID: fad90753-2022-4456-9b0a-c7e5b934e408
+          AZURE_CREDENTIALS_SECRET: ${{ secrets.AZURE_CREDENTIALS_SECRET }}
+          # unset the GitHub Action default JAVA_HOME to run with JDK 17
+          JAVA_HOME:
+          # output plain logs instead of json
+          TDR_LOG_APPENDER: 'Console-Standard'
```

Comment:
Reviewer note: this is the minimum number of environment variables actually required to run integration tests.